pkg/bgp/speaker/events.go

```diff
@@ -57,7 +57,7 @@ func (s *MetalLBSpeaker) run(ctx context.Context) {
 		// previous to this iteration, we processed an event
 		// which indicates the speaker should yield. shut
 		// it down.
-		if s.shutdown > 0 {
+		if s.shutdown > 0 { // atomic load not necessary, we are the only writer.
```

Comment:
Are these changes meant to be part of the 1st commit?

Reply:
[1]